resources/qml/Settings/SettingItem.qml

```diff
@@ -18,7 +18,6 @@ Item
     anchors.right: parent.right
 
     // To avoid overlapping with the scrollBars
```

Comment:
WHoops. Missed this line. Since the right anchor is removed, the comment should also be removed